jenkins/bootstrap_test.py

```diff
@@ -1403,10 +1409,33 @@ def CheckBootstrapYaml(self, path, check, suffix='suffix', use_json=False):
             self.assertTrue(job_name)
             self.assertEquals(job_name, real_job.get('job-name'))
 
+    def GetRealBootstrapJob(self, path, job):
```

Comment:
Should CheckBootstrapYaml use this function? It seems to duplicate a lot of functionality.